### PR TITLE
Add holidays for Finland

### PR DESCRIPTION
**Pull request by ConnySjoblom**
This PR adds Finnish holidays!

Would also be nice to see support for languages here. So having `languageCode()` on top of `countryCode()` for instance. Finland for instace has both Swedish (se_FI) and Finnish (fi_FI) as official languages.

### CODE REVIEW

File: src/Countries/Finland.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+use Carbon\CarbonImmutable;
+
+class Finland extends Country
+{
+    public function countryCode(): string
+    {
+        return 'fi';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```
Comment (Nielsvanpach) [inline src/Countries/Finland.php:14]: you can remove this

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! Could you rebase from main, so I can run the tests here?

**Comment by Nielsvanpach** [comment]
> Would also be nice to see support for languages here. So having languageCode() on top of countryCode() for instance. Finland for instace has both Swedish (se_FI) and Finnish (fi_FI) as official languages.

I'm open to a PR which adds language support. Would be best in a different PR.

File: src/Countries/Finland.php (hunk 1 above)
Comment (ConnySjoblom) [reply src/Countries/Finland.php]: Done

**Comment by ConnySjoblom** [comment]
git settings apparently wrong on second machine, will reopen.

**Pull request closed without merging**